src/Sylius/Component/Core/Calculator/ProductVariantPriceCalculator.php

```diff
@@ -25,7 +25,7 @@
     /**
      * {@inheritdoc}
      */
-    public function calculate(ProductVariantInterface $productVariant, array $context): int
+    public function calculate(ProductVariantInterface $productVariant, array $context): ?int
```

Comment:
Can we revert this change?

Reply:
Yes, after creating a new interface for original price .

Reply:
Yeah, this needs to be reverted in order to keep BC.